src/Avalonia.Diagnostics/Diagnostics/VisualExtensions.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.IO;
+using Avalonia.Controls;
+using Avalonia.Data;
+using Avalonia.Media.Imaging;
+using Avalonia.VisualTree;
+
+namespace Avalonia.Diagnostics
+{
+    public static class VisualExtensions
```

Comment:
Can you make it internal again? Users might start using it in their applications, while we want to keep Diagnostics package usable with DEBUG only configuration by default. So in release whole package can be removed.

Reply:
Plus, you already created base handler class, where it was used. So custom handlers would be easy to add anyway.